src/components/sidebar/InputContainer.tsx

```diff
@@ -110,6 +83,10 @@ class InputContainer extends React.Component<IInputProps, IInputState> {
                 />
             );
         }
+        const value = (typeof this.props.value === 'string'
```

Comment:
What is this supposed to do?

Reply:
This fixed that strange bug where empty field change caused some console error. I guess it was because before the value was casted as string / number, now it doesn't do that.

Code that caused that strange console error:
value={this.props.value !== null ? (this.props.value as string | number) : ''}